docs/docs/guides/tracking/tracing.mdx

```diff
@@ -748,4 +739,64 @@ Calculated Fields:
     * Duration
     * Status
 
+## FAQs
+
+### How do I disable tracing?
+
+**Environment Variable:**
+
+In situations where you want to unconditionally disable tracing for the entire program, you can set the environment variable `WEAVE_TRACING_ENABLED` to `false`.
```

Comment:
I don't think this is correct, should be `WEAVE_DISABLED=true`

Reply:
🤦 thank you